src/python/pants/backend/codegen/thrift/apache/python/subsystem.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from __future__ import annotations
+
+from typing import cast
+
+from pants.engine.rules import collect_rules
+from pants.option.subsystem import Subsystem
+
+
+class ThriftPythonSubsystem(Subsystem):
+    options_scope = "thrift-python"
+    help = "Options specific to generating Python from Thrift using Apache Thrift"
+
+    @classmethod
+    def register_options(cls, register):
+        super().register_options(register)
```

Comment:
Python doesn't need runtime dependencies?

Reply:
It may; I haven't figured that part out yet. Will leave to a follow-up PR once I can get an example repo written.